### ISSUE TITLE
[error] Missing task key: ensimeServerJars

### ISSUE THREAD

**Issue by fommil** [ensime/ensime-sbt#276]
this is actually a manifestation of #275 and following the Troubleshooting Guide (you **did** follow the guide, right?) and deleting your coursier cache for sonatype snapshots should clean it up.

```
rm -rf ~/.coursier/cache/v1/https/oss.sonatype.org/content/repositories/snapshots
```

**Comment by peterbecich** [comment]
This was solved for me by updating sbt-ensime from 1.12.0 to 1.12.12

**Comment by tlegutko** [comment]
I had similar error, as in issue name, but deleting sbt / coursier cache was of no help.

What worked for me was to also update `sbt-ensime`, but also to create `ensime.sbt` file in project root manually specyfing current ensime server version:
```
import org.ensime.EnsimeCoursierKeys._
ensimeServerVersion in ThisBuild := "2.0.0-M3"
```
Relevant info is described here in docs: http://ensime.org/build_tools/sbt/#customise.

Hope it helps if somebody stumbles upon it!
